packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -168,6 +170,8 @@ export class VaFileInput {
       this.vaChange.emit({ files: [] });
     }
     this.file = null;
+    this.updateStatusMessage(`File removed. No file selected.`);
```

Comment:
This was not getting read out in my testing. I think once it gets deleted, it may only be in the DOM briefly, and then it renders the component's initial state with "No files selected."